packages/bitcoin/src/validation/transaction-validation.ts

```diff
@@ -33,12 +33,12 @@ export function isValidBitcoinTransaction({
     throw new BitcoinError('InvalidNetworkAddress');
   }
 
-  if (!isBtcMinimumSpend({ amount })) {
+  if (!isBtcMinimumSpend(amount)) {
     throw new BitcoinError('InsufficientAmount');
   }
 
   const { amount: spendable } = calculateMaxSpend({ recipient, utxos, feeRate, feeRates });
-  if (!isBtcBalanceSufficient({ amount, spendable })) {
+  if (!isBtcBalanceSufficient({ desiredSpend: amount, maxSpend: spendable })) {
```

Comment:
I'm not sure about this arg renaming as now you need to specify here also.  Maybe it makes it easier to reason with and aligns with the extension?

Reply:
I borrowed desiredSpend & maxSpend from the extension yeah. I found the old naming to be very confusing to be honest. Missed this usage point though, will fix the naming here.

Reply:
Whatever you think is best I'm happy with. If I recall it took me a while to get my head around `desiredSpend ` and `maxSpend ` so I changed it to be `amount` and `spendable`.

I'm happy to go with the general consensus here